src/test/java/com/courtcanva/ccfranchise/utils/TestHelper.java

```diff
@@ -0,0 +1,93 @@
+package com.courtcanva.ccfranchise.utils;
+
+import com.courtcanva.ccfranchise.constants.AUState;
+import com.courtcanva.ccfranchise.dtos.FranchiseeAndStaffPostDto;
+import com.courtcanva.ccfranchise.dtos.FranchiseeGetDto;
+import com.courtcanva.ccfranchise.dtos.FranchiseePostDto;
+import com.courtcanva.ccfranchise.dtos.StaffGetDto;
+import com.courtcanva.ccfranchise.dtos.StaffPostDto;
+import com.courtcanva.ccfranchise.models.Franchisee;
+import com.courtcanva.ccfranchise.models.Staff;
+
+public class TestHelper {
```

Comment:
Now that it's more of a utility class, why don't you just declare all methods/functions as static to avoid instantiating it every time?

Reply:
yes, in this case, we can use static

Reply:
refactor it ，thx